src/account-info/AccountDetailsScreen.js

```diff
@@ -16,17 +16,21 @@ class AccountDetailsScreen extends Component {
     avatarUrl: string,
   }
 
+  componentWillMount() {
+    this.email = this.props.email;
```

Comment:
It is stored in 'current route' property and the current route changes at the point of animation. Thus during the sliding of the view it is undefined, and this is where the error came from.